releasenotes/notes/kubernetes-set-events-tag-according-to-involved-host-3f25d35c576ae5c1.yaml

```diff
@@ -0,0 +1,3 @@
+---
+features:
+  - Matching host tags for kubernetes events to the related hosts instead of the host collecting the events.
```

Comment:
The change is broader as the host tag matching is a side-effect of the hostname being transmitted, right?

Reply:
Sure, the wording here is a bit too approximate

Reply:
👍